### PR TITLE
Added indentation checking with Eslint

### LINKED ISSUE #185: Set up GitHub Action for Svelte and TypeScript linting
### What's your idea?

We need to establish a strict linter so that each contributor must appease the format requirements before merging into main.

- [x] <https://github.com/EthanThatOneKid/acmcsuf.com/pull/192>
- [ ] Prefer spaces exclusively over tabs.

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
- Created `seek-tabs.js` utility script: Accessible with `npm run tabs`, this script just checks to see if any of the changed files contain tabs. If so, they are logged to stdout. If any tabs are found, the process fails.
- Formatted entire codebase with `npm run format`.
- Added eslint linting to the `check_website.yaml` workflow.
- Updated `CONTRIBUTING.md` accordingly.

Intends to resolve #185.

### CODE REVIEW

**Comment by diamondburned** [comment]
We should consider if `eslint` can check for indentation first.

File: scripts/close-issue-channel.js [hunk 1]
```diff
@@ -33,130 +34,131 @@ startBot(async (client) => {
...
+  // eslint-disable-next-line no-constant-condition
+  while (true) {
+    const options = { limit: softLimit };
+    if (lastId) options.before = lastId;
+    const messages = await channel.messages.fetch(options);
+    if (messages === undefined) break;
+    result.push(...messages.values());
+    lastId = messages.last().id;
+    if (messages.size != softLimit || result >= limit) break;
+    await sleep(250); // Let's not hammer the API.
```
Comment (diamondburned) [inline scripts/close-issue-channel.js]: You don't have to worry about this. Discord.js will do it for you if it needs
to.

Comment (diamondburned) [reply scripts/close-issue-channel.js]: Agh, I forgot to check this in the previous PR. Maybe it can be a follow-up PR.

**Review by diamondburned — APPROVED** [review]

File: scripts/close-issue-channel.js (hunk 1 above)
Comment (EthanThatOneKid) [reply scripts/close-issue-channel.js]: Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/199/commits/d5ea2afb3fbbcb1abaec6ed71e5773c6fb08dd36>.

**Pull request merged**